File: Linguagem C/if_else_lista/Ex_012.c
//12) Faça um programa que leia as medidas dos 3 lados de um triângulo e mostre o tipo de triângulo:
//equilátero, isósceles ou escaleno. Imprima apenas o nome do triângulo, em letras minúsculas e sem acentos.
#include <stdio.h>
int main ()
{
  int a, b, c;

  printf("Digite o numero de a:\n");
  scanf("%d", &a);

  printf("Digite o numero de b:\n");
  scanf("%d", &b);

  printf("Digite o numero de c:\n");
  scanf("%d", &c);  
  
  if( a == b && a == c )
  {
    printf("equilatero");
  }   
    else    
        if(a != b && b != c)
        {
            printf("escaleno");
        }
            else    
                if((a == b && a != c) || (b == c && b != c))
                {
                    printf("isosceles");
                }  
return 0;
}

File: Python/004_Lista_if_else/Ex_012.py
"""
12) Faça um programa que leia as medidas dos 3 lados de um triângulo e mostre o tipo de triângulo:
equilátero, isósceles ou escaleno. Imprima apenas o nome do triângulo, em letras minúsculas e sem acentos.
"""

a = int(input("Digite o tamanho do lado A: "))
b = int(input("Digite o tamanho do lado B: "))
c = int(input("Digite o tamanho do lado C: "))

if a == b == c:
    print("Equilatero")
elif (a == b and a != c) or ( a == c and a!= b) or (b == c and b != a):
    print("Isosceles")
else: 
    print("Escaleno")